docs/schema/examples/etcher.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "etcher",
+  "lead": "Burn. Better.",
+  "description": "Burn images to SD cards & USB drives, safely and easily.",
+  "logo": "/etcher-io.png",
+  "screenshot": "/screenshot.png",
+  "features": [
+    {
+			"title": "Validated Burning",
+			"image": "sd",
+			"description": "No more writing images on corrupted cards and wondering why your device isn't booting."
+		},
+		{
+			"title": "Hard Drive Friendly",
+			"image": "hd",
+			"description": "Makes drive selection obvious to avoid wiping your entire hard-drive"
+		},
+		{
+			"title": "Beautiful Interface",
+			"image": "simple",
+			"description": "Who said burning SD cards has to be an eyesore."
+		},
+		{
+			"title": "Open Source",
+			"image": "open-source",
+			"description": "Made with JS, HTML, node.js and <a target='_blank' href='http://electron.atom.io/'>Electron</a>. Dive in and contribute!"
+		},
+		{
+			"title": "Cross Platform",
+			"image": "cross-platform",
+			"description": "Works for everyone,<br/>no more complicated install instructions."
+		},
+		{
+			"title": "More on the way",
+			"image": "coming-soon",
+			"description": "50% faster burns, simultaneous writing for multiple drives.<br/><a target='_blank' href='https://github.com/resin-io/etcher/milestones'>View our roadmap</a>"
+		}
+  ],
+  "analytics":  {
+    "debug": true,
+    "mixpanelToken": "0b4c23e0072d8dee7cf8ac7ab7c4a216",
+    "gaSite": "etcher.io",
+    "gaId": "UA-45671959-2",
+    "gosquaredId": "GSN-954701-N",
+    "prefix": "Etcher Website"
+  },
+  "nav": {
+    "header": [
+      {
+        "text": "Changelog",
```

Comment:
Should the changelog information be defined in this object?

Reply:
No I don't think so, this just describes the data used to make the heading component. The changelog info will be kept in a separate object. And used when pulling into the `/changelog` component. eg

```js
// pages/changelog.js
export default ({ entries }) => 
<div>
        {entries.map((entry, i) => {
          return <Entry {...entry.id} key={i} />;
        })}
</div>
```
It also gives the user an escape hatch as they may prefer to direct the user the releases page on github eg: 
```
{
     "text": "Changelog",
     "href" : "https://github.com/resin-io/landr/releases"
}
```